Docker build fails due to kafka

**Affected module**
Ingestion Framework

**Describe the bug**
confluent-kafka released a new version [1.9.0](https://pypi.org/project/confluent-kafka/1.9.0/#history) due to which the docker build fails with following error 

```
  Running setup.py install for confluent_kafka ... error
  error: subprocess-exited-with-error

  × Running setup.py install for confluent_kafka did not run successfully.
  │ exit code: 1
  ╰─> [281 lines of output]
      running install
      /usr/local/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build/lib.linux-aarch64-3.9
      creating build/lib.linux-aarch64-3.9/confluent_kafka
      copying src/confluent_kafka/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka
      copying src/confluent_kafka/error.py -> build/lib.linux-aarch64-3.9/confluent_kafka
      copying src/confluent_kafka/deserializing_consumer.py -> build/lib.linux-aarch64-3.9/confluent_kafka
      copying src/confluent_kafka/serializing_producer.py -> build/lib.linux-aarch64-3.9/confluent_kafka
      creating build/lib.linux-aarch64-3.9/confluent_kafka/kafkatest
      copying src/confluent_kafka/kafkatest/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka/kafkatest
      copying src/confluent_kafka/kafkatest/verifiable_client.py -> build/lib.linux-aarch64-3.9/confluent_kafka/kafkatest
      copying src/confluent_kafka/kafkatest/verifiable_consumer.py -> build/lib.linux-aarch64-3.9/confluent_kafka/kafkatest
      copying src/confluent_kafka/kafkatest/verifiable_producer.py -> build/lib.linux-aarch64-3.9/confluent_kafka/kafkatest
      creating build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      copying src/confluent_kafka/schema_registry/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      copying src/confluent_kafka/schema_registry/json_schema.py -> build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      copying src/confluent_kafka/schema_registry/avro.py -> build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      copying src/confluent_kafka/schema_registry/error.py -> build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      copying src/confluent_kafka/schema_registry/schema_registry_client.py -> build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      copying src/confluent_kafka/schema_registry/protobuf.py -> build/lib.linux-aarch64-3.9/confluent_kafka/schema_registry
      creating build/lib.linux-aarch64-3.9/confluent_kafka/avro
      copying src/confluent_kafka/avro/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka/avro
      copying src/confluent_kafka/avro/load.py -> build/lib.linux-aarch64-3.9/confluent_kafka/avro
      copying src/confluent_kafka/avro/error.py -> build/lib.linux-aarch64-3.9/confluent_kafka/avro
      copying src/confluent_kafka/avro/cached_schema_registry_client.py -> build/lib.linux-aarch64-3.9/confluent_kafka/avro
      creating build/lib.linux-aarch64-3.9/confluent_kafka/admin
      copying src/confluent_kafka/admin/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka/admin
      copying src/confluent_kafka/admin/_resource.py -> build/lib.linux-aarch64-3.9/confluent_kafka/admin
      copying src/confluent_kafka/admin/_acl.py -> build/lib.linux-aarch64-3.9/confluent_kafka/admin
      copying src/confluent_kafka/admin/_config.py -> build/lib.linux-aarch64-3.9/confluent_kafka/admin
      creating build/lib.linux-aarch64-3.9/confluent_kafka/serialization
      copying src/confluent_kafka/serialization/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka/serialization
      creating build/lib.linux-aarch64-3.9/confluent_kafka/avro/serializer
      copying src/confluent_kafka/avro/serializer/message_serializer.py -> build/lib.linux-aarch64-3.9/confluent_kafka/avro/serializer
      copying src/confluent_kafka/avro/serializer/__init__.py -> build/lib.linux-aarch64-3.9/confluent_kafka/avro/serializer
      running build_ext
      building 'confluent_kafka.cimpl' extension
      creating build/temp.linux-aarch64-3.9
      creating build/temp.linux-aarch64-3.9/tmp
      creating build/temp.linux-aarch64-3.9/tmp/pip-install-f__ouhf5
      creating build/temp.linux-aarch64-3.9/tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d
      creating build/temp.linux-aarch64-3.9/tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src
      creating build/temp.linux-aarch64-3.9/tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka
      creating build/temp.linux-aarch64-3.9/tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src
      gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c -o build/temp.linux-aarch64-3.9/tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.o
      In file included from /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:17:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/confluent_kafka.h:66:2: error: #error "confluent-kafka-python requires librdkafka v1.9.0 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"
         66 | #error "confluent-kafka-python requires librdkafka v1.9.0 or later. Install the latest version of librdkafka from the Confluent repositories, see http://docs.confluent.io/current/installation.html"
            |  ^~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:156:8: error: unknown type name ‘rd_kafka_AclBinding_t’
        156 | static rd_kafka_AclBinding_t *
            |        ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_py_to_c_AclBinding’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:162:9: error: unknown type name ‘rd_kafka_AclBinding_t’; did you mean ‘rd_kafka_AdminOptions_t’?
        162 |         rd_kafka_AclBinding_t *ret = NULL;
            |         ^~~~~~~~~~~~~~~~~~~~~
            |         rd_kafka_AdminOptions_t
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:172:27: warning: implicit declaration of function ‘rd_kafka_AclBinding_new’; did you mean ‘rd_kafka_AdminOptions_new’? [-Wimplicit-function-declaration]
        172 |                     ret = rd_kafka_AclBinding_new(restype, resname, \
            |                           ^~~~~~~~~~~~~~~~~~~~~~~
            |                           rd_kafka_AdminOptions_new
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:172:25: warning: assignment to ‘int *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
        172 |                     ret = rd_kafka_AclBinding_new(restype, resname, \
            |                         ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: At top level:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:185:8: error: unknown type name ‘rd_kafka_AclBindingFilter_t’
        185 | static rd_kafka_AclBindingFilter_t*
            |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_py_to_c_AclBindingFilter’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:192:9: error: unknown type name ‘rd_kafka_AclBindingFilter_t’
        192 |         rd_kafka_AclBindingFilter_t* ret = NULL;
            |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:201:27: warning: implicit declaration of function ‘rd_kafka_AclBindingFilter_new’ [-Wimplicit-function-declaration]
        201 |                     ret = rd_kafka_AclBindingFilter_new(restype, resname, \
            |                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:201:25: warning: assignment to ‘int *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
        201 |                     ret = rd_kafka_AclBindingFilter_new(restype, resname, \
            |                         ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_create_acls’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1045:9: error: unknown type name ‘rd_kafka_AclBinding_t’; did you mean ‘rd_kafka_AdminOptions_t’?
       1045 |         rd_kafka_AclBinding_t **c_objs = NULL;
            |         ^~~~~~~~~~~~~~~~~~~~~
            |         rd_kafka_AdminOptions_t
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1081:46: error: ‘RD_KAFKA_ADMIN_OP_CREATEACLS’ undeclared (first use in this function); did you mean ‘RD_KAFKA_ADMIN_OP_CREATETOPICS’?
       1081 |         c_options = Admin_options_to_c(self, RD_KAFKA_ADMIN_OP_CREATEACLS,
            |                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                              RD_KAFKA_ADMIN_OP_CREATETOPICS
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1081:46: note: each undeclared identifier is reported only once for each function it appears in
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1130:9: warning: implicit declaration of function ‘rd_kafka_CreateAcls’; did you mean ‘rd_kafka_CreateTopics’? [-Wimplicit-function-declaration]
       1130 |         rd_kafka_CreateAcls(self->rk, c_objs, cnt, c_options, rkqu);
            |         ^~~~~~~~~~~~~~~~~~~
            |         rd_kafka_CreateTopics
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1134:9: warning: implicit declaration of function ‘rd_kafka_AclBinding_destroy_array’; did you mean ‘rd_kafka_NewTopic_destroy_array’? [-Wimplicit-function-declaration]
       1134 |         rd_kafka_AclBinding_destroy_array(c_objs, cnt);
            |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |         rd_kafka_NewTopic_destroy_array
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_describe_acls’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1172:9: error: unknown type name ‘rd_kafka_AclBindingFilter_t’
       1172 |         rd_kafka_AclBindingFilter_t *c_obj = NULL;
            |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1200:46: error: ‘RD_KAFKA_ADMIN_OP_CREATEACLS’ undeclared (first use in this function); did you mean ‘RD_KAFKA_ADMIN_OP_CREATETOPICS’?
       1200 |         c_options = Admin_options_to_c(self, RD_KAFKA_ADMIN_OP_CREATEACLS,
            |                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                              RD_KAFKA_ADMIN_OP_CREATETOPICS
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1241:9: warning: implicit declaration of function ‘rd_kafka_DescribeAcls’; did you mean ‘rd_kafka_DescribeConfigs’? [-Wimplicit-function-declaration]
       1241 |         rd_kafka_DescribeAcls(self->rk, c_obj, c_options, rkqu);
            |         ^~~~~~~~~~~~~~~~~~~~~
            |         rd_kafka_DescribeConfigs
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1245:9: warning: implicit declaration of function ‘rd_kafka_AclBinding_destroy’; did you mean ‘rd_kafka_AdminOptions_destroy’? [-Wimplicit-function-declaration]
       1245 |         rd_kafka_AclBinding_destroy(c_obj);
            |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
            |         rd_kafka_AdminOptions_destroy
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_delete_acls’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1276:9: error: unknown type name ‘rd_kafka_AclBindingFilter_t’
       1276 |         rd_kafka_AclBindingFilter_t **c_objs = NULL;
            |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1312:46: error: ‘RD_KAFKA_ADMIN_OP_DELETEACLS’ undeclared (first use in this function); did you mean ‘RD_KAFKA_ADMIN_OP_DELETEGROUPS’?
       1312 |         c_options = Admin_options_to_c(self, RD_KAFKA_ADMIN_OP_DELETEACLS,
            |                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                              RD_KAFKA_ADMIN_OP_DELETEGROUPS
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1361:9: warning: implicit declaration of function ‘rd_kafka_DeleteAcls’; did you mean ‘rd_kafka_DeleteGroups’? [-Wimplicit-function-declaration]
       1361 |         rd_kafka_DeleteAcls(self->rk, c_objs, cnt, c_options, rkqu);
            |         ^~~~~~~~~~~~~~~~~~~
            |         rd_kafka_DeleteGroups
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: At top level:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1712:33: error: unknown type name ‘rd_kafka_AclBinding_t’
       1712 | Admin_c_AclBinding_to_py (const rd_kafka_AclBinding_t *c_acl_binding) {
            |                                 ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_c_AclBinding_to_py’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1725:38: warning: implicit declaration of function ‘rd_kafka_AclBinding_restype’ [-Wimplicit-function-declaration]
       1725 |                                      rd_kafka_AclBinding_restype(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1727:38: warning: implicit declaration of function ‘rd_kafka_AclBinding_name’ [-Wimplicit-function-declaration]
       1727 |                                      rd_kafka_AclBinding_name(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1727:38: warning: passing argument 3 of ‘cfl_PyDict_SetString’ makes pointer from integer without a cast [-Wint-conversion]
       1727 |                                      rd_kafka_AclBinding_name(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                      |
            |                                      int
      In file included from /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:17:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/confluent_kafka.h:325:74: note: expected ‘const char *’ but argument is of type ‘int’
        325 | void cfl_PyDict_SetString (PyObject *dict, const char *name, const char *val);
            |                                                              ~~~~~~~~~~~~^~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1729:33: warning: implicit declaration of function ‘rd_kafka_AclBinding_resource_pattern_type’ [-Wimplicit-function-declaration]
       1729 |                                 rd_kafka_AclBinding_resource_pattern_type(c_acl_binding));
            |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1731:38: warning: implicit declaration of function ‘rd_kafka_AclBinding_principal’ [-Wimplicit-function-declaration]
       1731 |                                      rd_kafka_AclBinding_principal(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1731:38: warning: passing argument 3 of ‘cfl_PyDict_SetString’ makes pointer from integer without a cast [-Wint-conversion]
       1731 |                                      rd_kafka_AclBinding_principal(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                      |
            |                                      int
      In file included from /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:17:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/confluent_kafka.h:325:74: note: expected ‘const char *’ but argument is of type ‘int’
        325 | void cfl_PyDict_SetString (PyObject *dict, const char *name, const char *val);
            |                                                              ~~~~~~~~~~~~^~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1733:38: warning: implicit declaration of function ‘rd_kafka_AclBinding_host’ [-Wimplicit-function-declaration]
       1733 |                                      rd_kafka_AclBinding_host(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1733:38: warning: passing argument 3 of ‘cfl_PyDict_SetString’ makes pointer from integer without a cast [-Wint-conversion]
       1733 |                                      rd_kafka_AclBinding_host(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                      |
            |                                      int
      In file included from /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:17:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/confluent_kafka.h:325:74: note: expected ‘const char *’ but argument is of type ‘int’
        325 | void cfl_PyDict_SetString (PyObject *dict, const char *name, const char *val);
            |                                                              ~~~~~~~~~~~~^~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1735:38: warning: implicit declaration of function ‘rd_kafka_AclBinding_operation’ [-Wimplicit-function-declaration]
       1735 |                                      rd_kafka_AclBinding_operation(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1737:38: warning: implicit declaration of function ‘rd_kafka_AclBinding_permission_type’ [-Wimplicit-function-declaration]
       1737 |                                      rd_kafka_AclBinding_permission_type(c_acl_binding));
            |                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: At top level:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1752:34: error: unknown type name ‘rd_kafka_AclBinding_t’
       1752 | Admin_c_AclBindings_to_py (const rd_kafka_AclBinding_t **c_acls,
            |                                  ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1777:33: error: unknown type name ‘rd_kafka_acl_result_t’
       1777 | Admin_c_acl_result_to_py (const rd_kafka_acl_result_t **c_result,
            |                                 ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_c_acl_result_to_py’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1786:51: warning: implicit declaration of function ‘rd_kafka_acl_result_error’; did you mean ‘rd_kafka_group_result_error’? [-Wimplicit-function-declaration]
       1786 |                 const rd_kafka_error_t *c_error = rd_kafka_acl_result_error(c_result[i]);
            |                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~
            |                                                   rd_kafka_group_result_error
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1786:51: warning: initialization of ‘const rd_kafka_error_t *’ {aka ‘const struct rd_kafka_error_s *’} from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: At top level:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1802:50: error: unknown type name ‘rd_kafka_DeleteAcls_result_response_t’
       1802 | Admin_c_DeleteAcls_result_responses_to_py (const rd_kafka_DeleteAcls_result_response_t **c_result_responses,
            |                                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_c_DeleteAcls_result_responses_to_py’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1804:15: error: unknown type name ‘rd_kafka_AclBinding_t’
       1804 |         const rd_kafka_AclBinding_t **c_matching_acls;
            |               ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1814:51: warning: implicit declaration of function ‘rd_kafka_DeleteAcls_result_response_error’; did you mean ‘Admin_c_DeleteAcls_result_responses_to_py’? [-Wimplicit-function-declaration]
       1814 |                 const rd_kafka_error_t *c_error = rd_kafka_DeleteAcls_result_response_error(c_result_responses[i]);
            |                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                                   Admin_c_DeleteAcls_result_responses_to_py
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1814:51: warning: initialization of ‘const rd_kafka_error_t *’ {aka ‘const struct rd_kafka_error_s *’} from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1822:43: warning: implicit declaration of function ‘rd_kafka_DeleteAcls_result_response_matching_acls’ [-Wimplicit-function-declaration]
       1822 |                         c_matching_acls = rd_kafka_DeleteAcls_result_response_matching_acls(
            |                                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1822:41: warning: assignment to ‘const int **’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
       1822 |                         c_matching_acls = rd_kafka_DeleteAcls_result_response_matching_acls(
            |                                         ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c: In function ‘Admin_background_event_cb’:
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1941:14: error: ‘RD_KAFKA_EVENT_CREATEACLS_RESULT’ undeclared (first use in this function); did you mean ‘RD_KAFKA_EVENT_CREATETOPICS_RESULT’?
       1941 |         case RD_KAFKA_EVENT_CREATEACLS_RESULT:
            |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |              RD_KAFKA_EVENT_CREATETOPICS_RESULT
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1943:23: error: unknown type name ‘rd_kafka_acl_result_t’
       1943 |                 const rd_kafka_acl_result_t **c_acl_results;
            |                       ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1946:33: warning: implicit declaration of function ‘rd_kafka_CreateAcls_result_acls’; did you mean ‘rd_kafka_CreateTopics_result_topics’? [-Wimplicit-function-declaration]
       1946 |                 c_acl_results = rd_kafka_CreateAcls_result_acls(
            |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                 rd_kafka_CreateTopics_result_topics
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1947:25: warning: implicit declaration of function ‘rd_kafka_event_CreateAcls_result’; did you mean ‘rd_kafka_event_CreateTopics_result’? [-Wimplicit-function-declaration]
       1947 |                         rd_kafka_event_CreateAcls_result(rkev),
            |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                         rd_kafka_event_CreateTopics_result
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1946:31: warning: assignment to ‘const int **’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
       1946 |                 c_acl_results = rd_kafka_CreateAcls_result_acls(
            |                               ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1956:14: error: ‘RD_KAFKA_EVENT_DESCRIBEACLS_RESULT’ undeclared (first use in this function); did you mean ‘RD_KAFKA_EVENT_DESCRIBECONFIGS_RESULT’?
       1956 |         case RD_KAFKA_EVENT_DESCRIBEACLS_RESULT:
            |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |              RD_KAFKA_EVENT_DESCRIBECONFIGS_RESULT
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1958:23: error: unknown type name ‘rd_kafka_DescribeAcls_result_t’
       1958 |                 const rd_kafka_DescribeAcls_result_t *c_acl_result;
            |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1959:23: error: unknown type name ‘rd_kafka_AclBinding_t’
       1959 |                 const rd_kafka_AclBinding_t **c_acls;
            |                       ^~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1962:32: warning: implicit declaration of function ‘rd_kafka_event_DescribeAcls_result’; did you mean ‘rd_kafka_event_DescribeConfigs_result’? [-Wimplicit-function-declaration]
       1962 |                 c_acl_result = rd_kafka_event_DescribeAcls_result(rkev);
            |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                rd_kafka_event_DescribeConfigs_result
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1962:30: warning: assignment to ‘const int *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
       1962 |                 c_acl_result = rd_kafka_event_DescribeAcls_result(rkev);
            |                              ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1965:26: warning: implicit declaration of function ‘rd_kafka_DescribeAcls_result_acls’; did you mean ‘rd_kafka_DescribeConfigs_result_resources’? [-Wimplicit-function-declaration]
       1965 |                 c_acls = rd_kafka_DescribeAcls_result_acls(
            |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                          rd_kafka_DescribeConfigs_result_resources
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1965:24: warning: assignment to ‘const int **’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
       1965 |                 c_acls = rd_kafka_DescribeAcls_result_acls(
            |                        ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1982:14: error: ‘RD_KAFKA_EVENT_DELETEACLS_RESULT’ undeclared (first use in this function); did you mean ‘RD_KAFKA_EVENT_DELETETOPICS_RESULT’?
       1982 |         case RD_KAFKA_EVENT_DELETEACLS_RESULT:
            |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |              RD_KAFKA_EVENT_DELETETOPICS_RESULT
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1984:23: error: unknown type name ‘rd_kafka_DeleteAcls_result_t’
       1984 |                 const rd_kafka_DeleteAcls_result_t *c_acl_result;
            |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1985:23: error: unknown type name ‘rd_kafka_DeleteAcls_result_response_t’
       1985 |                 const rd_kafka_DeleteAcls_result_response_t **c_acl_result_responses;
            |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1988:32: warning: implicit declaration of function ‘rd_kafka_event_DeleteAcls_result’; did you mean ‘rd_kafka_event_DeleteGroups_result’? [-Wimplicit-function-declaration]
       1988 |                 c_acl_result = rd_kafka_event_DeleteAcls_result(rkev);
            |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                rd_kafka_event_DeleteGroups_result
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1988:30: warning: assignment to ‘const int *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
       1988 |                 c_acl_result = rd_kafka_event_DeleteAcls_result(rkev);
            |                              ^
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1990:42: warning: implicit declaration of function ‘rd_kafka_DeleteAcls_result_responses’; did you mean ‘rd_kafka_DeleteGroups_result_groups’? [-Wimplicit-function-declaration]
       1990 |                 c_acl_result_responses = rd_kafka_DeleteAcls_result_responses(
            |                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            |                                          rd_kafka_DeleteGroups_result_groups
      /tmp/pip-install-f__ouhf5/confluent-kafka_183aa92ee8594a1ebde2b9db7b15c81d/src/confluent_kafka/src/Admin.c:1990:40: warning: assignment to ‘const int **’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
       1990 |                 c_acl_result_responses = rd_kafka_DeleteAcls_result_responses(
            |                                        ^
      error: command '/usr/bin/gcc' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  WARNING: No metadata found in /usr/local/lib/python3.9/site-packages
  Rolling back uninstall of confluent-kafka
  Moving to /usr/local/LICENSE.txt
   from /tmp/pip-uninstall-nzbxtv7p/LICENSE.txt
  Moving to /usr/local/lib/python3.9/site-packages/confluent_kafka-1.8.2.dist-info/
   from /usr/local/lib/python3.9/site-packages/~onfluent_kafka-1.8.2.dist-info
  Moving to /usr/local/lib/python3.9/site-packages/confluent_kafka/
   from /usr/local/lib/python3.9/site-packages/~onfluent_kafka
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> confluent_kafka

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
WARNING: You are using pip version 22.0.4; however, version 22.1.2 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
```


**To Reproduce**
build the openmetadata docker

--- a/ingestion/setup.py
+++ b/ingestion/setup.py
@@ -102,7 +102,7 @@ plugins: Dict[str, Set[str]] = {
         "presto-types-parser==0.0.2",
     },
     "kafka": {
-        "confluent_kafka>=1.5.0",
+        "confluent_kafka==1.8.2",
         "fastavro>=1.2.0",
         "avro-python3",
         "confluent_avro",

